website/pages/docs/k8s/helm.mdx

```diff
@@ -730,9 +765,118 @@ and consider if they're appropriate for your deployment.
         }
       ```
 
+- `meshGateway` ((#v-meshgateway)) - Configure mesh gateways.
+
+  - `enabled` ((#v-meshgateway-enabled)) (`boolean: true`) - If mesh gateways are enabled, a Deployment will be created that runs
+    gateways and Consul Connect will be configured to use gateways.
+    See https://www.consul.io/docs/connect/mesh_gateway.html
+    Requirements: Consul 1.6.0+ and consul-k8s 0.9.0+ if using
+    `global.acls.manageSystemACLs`.
+
+  - `globalMode` ((#v-meshgateway-globalmode)) (`string: "local"`) - Globally configure which mode the gateway should run in.
+    Can be set to either `"remote"`, `"local"`, `"none"` or `""` or `null`.
+    See https://consul.io/docs/connect/mesh_gateway.html#modes-of-operation for
+    a description of each mode.
+    If set to anything other than `""` or `null`, `connectInject.centralConfig.enabled`
+    should be set to true so that the global config will actually be used.
+    If set to the empty string, no global default will be set and the gateway mode
+    will need to be set individually for each service.
+
+  - `replicas` ((#v-meshgateway-replicas)) (`int: 2`) - Number of replicas for the Deployment.
+
+  - `wanAddress` ((#v-meshgateway-wanaddress)) - What gets registered as WAN (wide area network) address for the gateway.
+
+    - `source` ((#v-meshgateway-wanaddress-source)) (`string: "Service"`) - source configures where to retrieve the WAN address (and possibly port)
+        for the mesh gateway from.
+        Can be set to either: `Service`, `NodeIP`, `NodeName` or `Static`. See the behaviour of each below:
```

Comment:
Nit: 🇺🇸 (I think we use American English on our website, so it'll be nice to stay consistent).
```suggestion
        Can be set to either: `Service`, `NodeIP`, `NodeName` or `Static`. See the behavior of each below:
```